app-frontend/config/webpack/global.js

```diff
@@ -267,7 +277,7 @@ module.exports = function (_path) {
                 'BUILDCONFIG': {
                     APP_NAME: JSON.stringify('Raster Foundry'),
                     BASEMAPS: basemaps,
-                    API_HOST: JSON.stringify(''),
+                    API_HOST: JSON.stringify('https://app.staging.rasterfoundry.com'),
```

Comment:
this shouldn't be changed - you can use the overrides file to test locally

Reply:
Derp, I forgot that wasn't gitignored; will remove.